src/test/java/com/wcc/platform/factories/SetupFactories.java

```diff
@@ -252,13 +249,12 @@ public static PageSection createPageSectionTest(final String title) {
   /**
    * Factory test for repository file section to delete the content of the file used for testing.
    */
-  public static void emptyFileRepositoryContent(final File testFile) {
-    try {
-      if (testFile.exists()) {
-        OBJECT_MAPPER.writeValue(testFile, new ArrayList<>());
+  public static void deleteFile(final File file) {
+    if (file.exists()) {
+      if (!file.delete()) {
+        final String msg = "Error deleting file " + file.getAbsolutePath();
+        throw new RuntimeException(msg);
```

Comment:
Throw exception (Runtime or better FileRepositoryException) or this is not needed?